dvc/exceptions.py

```diff
@@ -212,6 +212,8 @@ def __init__(self):
 
 class StageFileCorruptedError(DvcException):
     def __init__(self, path):
+        if os.path.isabs(path) and path.startswith(os.getcwd()):
```

Comment:
No need for these checks, if path is relative, it will return it anyway. Also, `path.startswith(os.getcwd())` doesn't seem to make sense at all :) Unless I'm missing something.

Reply:
Hmm, creating this PR I didn't take under account that that is not possible to create stage file outside of DVC repo. In such case startswith part is pointless.